solutions/slackbot/api/_constants.js

```diff
@@ -0,0 +1,12 @@
+import { Redis } from '@upstash/redis'
+
+export const token = process.env.SLACK_BOT_TOKEN
+export const signingSecret = process.env.SLACK_SIGNING_SECRET
+
+const redisURL = process.env.UPSTASH_REDIS_REST_URL
```

Comment:
let's create a .env.example with those to help people prefill their dev environment when cloning